templates/customize-class.html

```diff
@@ -1,92 +1,46 @@
-{% extends "auth.html" %}
-
+{% extends hx_layout %}
```

Comment:
Is `customize-class.html` intended to be retrieved via HTMX? If not, `hx_layout` doesn't do anything.

If yes, can we do something with the naming convention to indicate that? `hx-<name>.html` or somesuch?

Reply:
Ohhh, perfect! I included it because I thought that its use was that a page that uses HTMX calls needed to include it. Will remove it then!